src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -2525,7 +2525,7 @@ void java_lang_Throwable::fill_in_stack_trace(Handle throwable, const methodHand
       if (skip_hidden) {
         if (total_count == 0) {
           // The top frame will be hidden from the stack trace.
-          bt.set_has_hidden_top_frame(CHECK);
+          bt.set_has_hidden_top_frame();
```

Comment:
To be clear: no exception can become pending due to this call, so no need for CHECK.

Reply:
Correct.  I think it used to allocate an object so could throw OOM but that was changed.